src/main/kotlin/br/com/simpli/sql/Query.kt

```diff
@@ -344,29 +358,12 @@ open class Query {
      *
      * SELECT COUNT(column) FROM table GROUP BY column, other
      */
-    open fun groupBy(vararg columns: String) = groupByRaw(columns.joinToString())
-
-    /**
-     * simply adds "HAVING" in the query
-     *
-     * Query("SELECT COUNT(column) AS countt FROM table GROUP BY column").having("countt > 3")
-     *
-     * SELECT COUNT(column) AS countt FROM table GROUP BY column HAVING countt > 3
-     */
-    open fun having(str: String?, vararg param: Any?) = raw("HAVING $str", *param)
```

Comment:
no one has ever used this and is a pain to maintain this feature, we can implement it back in the future